pkg/logging/logging_test.go

```diff
@@ -0,0 +1,39 @@
+package logging
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func FakeHandler(w http.ResponseWriter, r *http.Request) {
+	w.WriteHeader(http.StatusOK)
+	w.Header().Set("Content-Type", "application/json")
+	fmt.Fprint(w, `{"retval": "done"}`)
+}
+
+func TestLoggingMiddleware(t *testing.T) {
+	req, err := http.NewRequest("GET", "/some-path", nil)
+	assert.NoError(t, err)
+
+	rr := httptest.NewRecorder()
+	handler := http.HandlerFunc(FakeHandler)
+
+	var capture bytes.Buffer
+
+	writer := bufio.NewWriter(&capture)
+	log.SetOutput(writer)
```

Comment:
I'm not sure it's a great idea to have middleware use the global default logger. This creates an implicit dependency and requires users to include a separate import (`logrus`) in order to change logging configuration.

Would it be better for the middleware to use it's own logrus instance and accept options to configure things like writers, log format, etc.?

Reply:
I'll see what options logrus has for this.  I am ok being opinionated about what logging framework we use in everything `kanopy-platform`.

Reply:
Then is should be more specific about what it is in the API design. Such as, `logging.NewLogrus()`, etc.

Reply:
Also like this, of NewLogrus.   I keep forgetting that logrus is in maintenance mode.  I think we may want to evaluate a different logging framework at some point.